openerp/tools/translate.py

```diff
@@ -329,6 +329,8 @@ def next(self):
                 elif line.startswith('#,') and (line[2:].strip() == 'fuzzy'):
                     fuzzy = True
                 line = self.lines.pop(0).strip()
+            if 0 == len(self.lines):
```

Comment:
The recommendation is generally to write better code when you change it (if you can), but not mix new fixes with refactoring/rewriting of working code.